es6-destructuring/main.js

```diff
@@ -0,0 +1,62 @@
+const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+
+const {
+  title,
+  author,
+  libraryID
+} = book1;
+
+console.log('The title of the book is ' + title + ', the author is ' +
+author + ', and the library id is ' + libraryID);
+
+const book2 = {
+  title: 'Les Fleurs du mal',
+  author: 'Charles Baudelaire',
+  libraryID: 2345
+};
+
+const {
+  title: bookTitle,
+  author: bookAuthor,
+  libraryID: bookLibraryID
+} = book2;
+
+console.log('The title of the book is ' + bookTitle + ', the author is ' +
+  bookAuthor + ', and the library id is ' + bookLibraryID);
+
+const library = [
+  {
+    title: 'The Road Ahead',
+    author: 'Bill Gates',
+    libraryID: 1254
+  },
+  {
+    title: 'Walter Isaacson',
+    author: 'Steve Jobs',
+    libraryID: 4264
+  },
+  {
+    title: 'Mockingjay: The Final Book of The Hunger Games',
+    author: 'Suzanne Collins',
+    libraryID: 3245
+  },
+  {
+    title: "Dead Dead Demon's De De De De Destruction",
+    author: 'Inio Asano',
+    libraryID: 1233
+  }
+];
+
+const [book3, book4, book5] = library;
+
+console.log('book3: ', book3);
+console.log('book4: ', book4);
+console.log('book5: ', book5);
+
+const [fourth] = library;
```

Comment:
small update needed here. This is assigning the same object as `book3` above. Destructuring does not change the original array, so using only one value here is selecting the first item of the array.